fastcrypto-cli/src/vdf.rs

```diff
@@ -223,22 +241,22 @@ mod tests {
             bit_length: 1024,
         }))
         .unwrap();
-        let expected = "Discriminant: ff6cb04c161319209d438b6f016a9c3703b69fef3bb701550eb556a7b2dfec8676677282f2dd06c5688c51439c59e5e1f9efe8305df1957d6b7bf3433493668680e8b8bb05262cbdf4d020dafa8d5a3433199b8b53f6d487b3f37a4ab59493f050d1e2b535b7e9be19c0201055c0d7a07db3aaa67fe0eed63b63d86558668a27".to_string();
+        let expected = "Discriminant: 8101ff00934fb3e9ece6df62bc7490fe9563c8fc496010c448feaaf14aa9584d20137989988d7d0d22f93a9773aebc63a61a1e061017cfa20e6a8294840cbccb6c99797f174744fad9d3420b2fdf250572a5cbcce66474ac092b784c0c85b54a6b6c0faf2e1d4aca481641e63fdfefaa3f285f824c5559801f1129c49c279aa79975d9".to_string();
```

Comment:
ooc, as this is 1024 bits, did we check against any Chia discriminant related test vector?

Reply:
I've added a unit test that compares with chia.